core/src/main/java/org/apache/calcite/sql/SqlIfStmt.java

```diff
@@ -41,6 +44,10 @@ public SqlIfStmt(final SqlParserPos pos,
       final SqlNodeList conditionalStmtListPairs,
       final SqlNodeList elseStmtList) {
     super(pos);
+    if (conditionalStmtListPairs == null) {
```

Comment:
You should just be able to use Objects.requireNonNull() for this.

Reply:
Sounds good. Added here https://github.com/googleinterns/calcite/pull/245/commits/0a11397894f8c800ebd81980b290f4a2dda11c7f